pkg/utils/utils.go

```diff
@@ -81,3 +82,19 @@ func FetchKey(keyName string) string {
 	f := reflect.Indirect(r).FieldByName(keyName)
 	return f.String()
 }
+
+func MaxWidth(a, b int) int {
```

Comment:
you can use `math.Max`

Reply:
`math.Max` takes float64 as input. there is no inbuilt max function that takes integers as input